packages/enclave/src/controllers/eth/wallets.controller.ts

```diff
@@ -360,6 +360,25 @@ export default class WalletsController
     );
   }
 
+  private async resendMasterWalletTransaction(
+    req: express.Request
+  ): Promise<EthTransaction> {
+    const masterWallet = await this.getMasterWalletById(
+      req.sdk,
+      req.params.masterWalletId
```

Comment:
In document, `masterWalletId` is delivered as request body, not parameter. Please check this :)

Reply:
added that part. 
```
    this.router.post(
      `${this.path}/:masterWalletId/transactions/resend`,
      this.promiseWrapper(this.resendMasterWalletTransaction, 201)
    );
```
i guess /:masterWalletId/ Route parameters.